scripts/github-indent-warnings.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+import sys, re
+
+re_file = r'^diff --git a/(\S\S*)\s.*$'
+re_line = r'^@@ -(\d\d*)\D.*@@.*$'
+
+if __name__ == '__main__':
+    if len(sys.argv) != 1 and len(sys.argv) != 2:
+        print(f'usage: {sys.argv[0]} <path/to/file>')
+        print(f'usage: <command> | {sys.argv[0]}')
+        exit(1)
+
+    with open(sys.stdin.fileno() if len(sys.argv) == 1 else sys.argv[1], 'r') as fi:
```

Comment:
Would it be possible to split this into multiple lines? For example, something like the following:
```Python
input_file = sys.stdin.fileno()
if len(sys.argv) == 1:
    input_file = sys.argv[1]

with open(input_file, 'r') as fi:
   ...
```
([*simple is better than complex*](https://peps.python.org/pep-0020/))

Reply:
Sure, you are right, done.